pkg/apiserver/server.go

```diff
@@ -145,6 +167,7 @@ func Start(params *APIServerParams) {
 			log.Println("Failed to run automated installs:", err)
 		}
 	}
+	logger.Debug("automated airgap installs are completed")
```

Comment:
i would be confused if i saw this in an online install

Reply:
```suggestion
	logger.Debug("automated airgap installs are completed if requested")
```